CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [1.24.0] - 2024-02-07
+
+### Added
+- Add `hazardDatasets` field to TornadoDataset, TornadoModel and EarthquakeModel class [#213](https://github.com/IN-CORE/incore-services/issues/213)
+- Renaming option for uploading geoserver layers [#259](https://github.com/IN-CORE/incore-services/issues/259)
+- GeoPakage handling in services  [#205](https://github.com/IN-CORE/incore-services/issues/205)
```

Comment:
Geopackage is spelled incorrectly and issue #213 is listed twice.

Reply:
Fixed typo and removed duplicated entry